src/aliases/store/postgres/postgres.go

```diff
@@ -0,0 +1,58 @@
+package aliaspg
```

Comment:
You need to follow the same structure as the `stores` database folder, you don't need a folder called `database` and your root is `store` (because the data layer of aliases is only the DB):

- store
- - database.go
- - postgres
- - - alias.go
- - - database.go
- - models
- - - alias.go
- - mock
- - - database.go

Even if you only have one Repository/Data agent for now, it's better to have a `Database` interface that has an `Alias()` function for consistency and scalability.

Reply:
I've organized as presented here.
I'd like to discuss the way we create the Database interface, I think there would be a better way. I'll add it to the document of our common approach.